Source/Blazorise.Bootstrap/wwwroot/blazorise.bootstrap.css

```diff
@@ -762,6 +762,11 @@ input[readonly=readonly][type=range]::-ms-thumb {
   display: contents;
 }
 
+.b-required:before {
+  content: "* ";
+  color: #dc3545;
```

Comment:
```suggestion
  color: var(--b-theme-danger, #dc3545);
```

Reply:
You're looking at compiled css, either way makes sense to prioritize the b-theme var.
`    color: var(--b-theme-danger, $danger);`